docker/incremental_load.sh.tpl

```diff
@@ -237,3 +237,7 @@ function read_variables() {
 # List of 'tag_layer' statements for all tags.
 # This generated and injected by docker_*.
 %{tag_statements}
+
+# An optional "docker run" statement for invoking a loaded container.
+# This generated and injected by docker_*.
+%{run_statements}
```

Comment:
The arguments pass through, and so I don't want to wade into the morass of distinguishing between user arguments and script arguments (or worse: wrapping script arguments).  My ultimate goal would be to have `legacy_XXX` opt you OUT (opt in by default).  I have considered adding back the `:foo.load` target, but Bazel's story around `runfiles` for runnable implicit targets leaves a lot to be desired, which is why I didn't do both here.